src/main/java/edu/greenblitz/robotName/subsystems/arm/wrist/srxWrist/SRXWrist.java

```diff
@@ -1,31 +1,32 @@
 package edu.greenblitz.robotName.subsystems.arm.wrist.srxWrist;
 
-import com.ctre.phoenix.motorcontrol.FeedbackDevice;
-import com.ctre.phoenix.motorcontrol.NeutralMode;
-import com.ctre.phoenix.motorcontrol.TalonSRXControlMode;
+import com.ctre.phoenix.motorcontrol.*;
 import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import com.revrobotics.CANSparkBase;
 import com.revrobotics.CANSparkMax;
 import edu.greenblitz.robotName.RobotConstants;
 import edu.greenblitz.robotName.subsystems.Battery;
+import edu.greenblitz.robotName.subsystems.arm.roller.Roller;
 import edu.greenblitz.robotName.subsystems.arm.wrist.IWrist;
 import edu.greenblitz.robotName.subsystems.arm.wrist.WristInputsAutoLogged;
 import edu.greenblitz.robotName.utils.Conversions;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 
 public class SRXWrist implements IWrist {
 
     private TalonSRX motor;
 
     public SRXWrist() {
         motor = new TalonSRX(SRXWristConstants.MOTOR_ID);
+
+        motor.configAllSettings(SRXWristConstants.TALON_SRX_CONFIGURATION);
         motor.configSelectedFeedbackSensor(
-                FeedbackDevice.CTRE_MagEncoder_Absolute,
+                FeedbackDevice.CTRE_MagEncoder_Relative,
```

Comment:
We need to sample the absolute position from the encoder and reset the relative position based on the absolute result (and the Noam needs to put the wrist somewhere near where it needs to start at)